src/test/testing-library/AuthFormFunctions.test.tsx

```
@@ -0,0 +1,371 @@
+    act(() => {
+      if (error instanceof Error) {
+        errorDetails = error.message;
+      }
+    });
+
+    expect(errorDetails).toBe('Something went wrong');
+  });
+
+  test('handles unknown error', () => {
+    const error: unknown = 42;
+    let errorDetails: string = '';
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)